builder/docker/communicator.go

```diff
@@ -194,8 +194,34 @@ func (c *Communicator) UploadDir(dst string, src string, exclude []string) error
 	return nil
 }
 
+// Download pulls a file out of a container using `docker cp`. We have a source
```

Comment:
`docker cp` supports directories, but the communicator Download interface in packer does not. So this must be a file on the filesystem (text, binary, whatever).